contracts/truefi/TrueRatingAgencyV2.sol

```diff
@@ -392,7 +392,8 @@ contract TrueRatingAgencyV2 is ITrueRatingAgencyV2, Ownable {
         // Update total and rater's TRU rewards for a loan
         uint256 totalReward = 0;
         uint256 ratersReward = 0;
-        if (loans[id].reward == 0) {
+        // rewards on older USDC loans were from 1 to 4 due to a bug with decimal conversion
```

Comment:
Let's expand this comment and describe conditions for when we can remove the special case logic. Something like this?

```suggestion
        // TODO remove this special case logic after the 3 old USDC loans mentioned in PR #932 have been fixed.
        // Rewards on these loans were from 1 to 4 due to a bug with decimal conversion fixed in PR #685.
```